Backpack/src/main/java/net/skyscanner/backpack/calendar2/list/CalendarCellDayHolder.kt

```diff
@@ -95,8 +95,12 @@ internal class CalendarCellDayHolder(
                     is CellLabel.Text -> {
                         label.isVisible = cellLabel.text.isNotEmpty()
                         label.setTextColor(labelColor(model.info.status))
+                        icon.visibility = View.GONE
```

Comment:
Ensures that the icon gets hidden when the text is being rendered. Failing to hide the icon could show the text and icon as labels.